--- bash_completion	2005-05-31 20:21:36.000000000 +0300
+++ /tmp/bash_completion	2005-05-31 20:21:09.000000000 +0300
@@ -229,6 +229,11 @@
 		COMPREPLY=( $( command ls \
 			/etc/sysconfig/network/ifcfg-* | \
 			sed -ne 's|.*ifcfg-\('$cur'.*\)|\1|p' ) )
+	elif [ -f /etc/pld-release ]; then
+		# PLD Linux
+		COMPREPLY=( $( command ls -B \
+			/etc/sysconfig/interfaces | \
+			sed -ne 's|.*ifcfg-\('$cur'.*\)|\1|p' ) )
 	else
 		# Assume Red Hat
 		COMPREPLY=( $( command ls \
