--- bash_completion/bash_completion~	2006-03-01 21:23:16.000000000 +0200
+++ bash_completion/bash_completion	2006-03-01 21:28:37.000000000 +0200
@@ -1742,7 +1742,7 @@
 	if [ -r /var/log/rpmpkgs -a /var/log/rpmpkgs -nt /var/lib/rpm/Packages ]; then
 		# using RHL 7.2 or later - this is quicker than querying the DB
 		COMPREPLY=( $( sed -ne \
-		's|^\('$cur'.*\)-[0-9a-zA-Z._]\+-[0-9a-z.@]\+.*\.rpm$|\1|p' \
+ 		's|^\('$cur'.*\)\.rpm$|\1|p' \
 				/var/log/rpmpkgs ) )
 	else
 		nodig=""
@@ -1757,8 +1757,7 @@
 			nosig="--nosignature"
 		fi
 
-		COMPREPLY=( $( rpm -qa $nodig $nosig | sed -ne \
-		's|^\('$cur'.*\)-[0-9a-zA-Z._]\+-[0-9a-z.@]\+$|\1|p' ) )
+ 		COMPREPLY=( $( rpm -qa $nodig $nosig --qf '%{name}-%{version}-%{release}.%{arch}\n' "$cur*" ) )
 	fi
 }
 
