--- bash-completion-2.11/bash_completion.orig	2021-03-06 22:24:43.862534689 +0100
+++ bash-completion-2.11/bash_completion	2021-03-07 06:44:50.513308049 +0100
@@ -1801,7 +1801,7 @@ _known_hosts_real()
 
 } # _known_hosts_real()
 complete -F _known_hosts traceroute traceroute6 \
-    fping fping6 telnet rsh rlogin ftp dig mtr ssh-installkeys showmount
+    fping fping6 telnet rsh rlogin ftp dig ssh-installkeys showmount
 
 # This meta-cd function observes the CDPATH variable, so that cd additionally
 # completes on directories under those specified in CDPATH.
