diff -ruNp bash-completion-2.8.orig/bash_completion bash-completion-2.8/bash_completion
--- bash-completion-2.8.orig/bash_completion	2018-03-17 09:25:59.000000000 +0100
+++ bash-completion-2.8/bash_completion	2018-06-24 00:45:30.480896640 +0200
@@ -1659,7 +1659,7 @@ _known_hosts_real()
 
 } # _known_hosts_real()
 complete -F _known_hosts traceroute traceroute6 tracepath tracepath6 \
-    fping fping6 telnet rsh rlogin ftp dig mtr ssh-installkeys showmount
+    fping fping6 telnet rsh rlogin ftp dig ssh-installkeys showmount
 
 # This meta-cd function observes the CDPATH variable, so that cd additionally
 # completes on directories under those specified in CDPATH.
