--- bash-completion-1.1/bash_completion~	2009-10-10 16:45:12.000000000 +0300
+++ bash-completion-1.1/bash_completion	2009-11-19 13:02:18.453957906 +0200
@@ -757,7 +757,7 @@
             _services
         else
             COMPREPLY=( $( compgen -W '`sed -ne "y/|/ /; \
-            s/^.*\(U\|msg_u\)sage.*{\(.*\)}.*$/\1/p" \
+            s/^.*\(U\|msg_u\)sage.*{\(.*\)}.*$/\2/p" \
             $sysvdir/${prev##*/} 2>/dev/null`' -- "$cur" ) )
         fi
 
