--- bash-completion-1.2/completions/pear~	2010-08-23 20:41:58.000000000 +0300
+++ bash-completion-1.2/completions/pear	2010-10-19 22:52:26.008412279 +0300
@@ -106,6 +106,13 @@
             _filedir '@(tgz)'
             return 0
             ;;
+        make-rpm-spec)
+            if [[ "${cur}" == -* ]] ; then
+                COMPREPLY=( $(compgen -W "-t -o -r -p -d --spec-template --output --rpm-release --rpm-pkgname --rpm-depname" -- ${cur}) )
+            fi
+            _filedir '@(tgz)'
+            return 0
+            ;;
         package|pickle)
             if [[ "${cur}" == -* ]] ; then
                 COMPREPLY=( $(compgen -W "-Z -n --nocompress --showname" -- ${cur}) )
